apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/proto/rda/v1/fiss/fiss_adm_type_code.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package gov.cms.mpsm.rda.v1.fiss;
+
+option java_multiple_files = true;
+
+import "rda/v1/enum_options.proto";
+
+// FISS Adm Type Code
+enum FissAdmTypeCode {
+  ADM_TYPE_0 = 0 	        	  [(string_value) = "0"];
+  ADM_TYPE_EMERGENCY = 1      [(string_value) = "1"];
+  ADM_TYPE_URGENT = 2   		  [(string_value) = "2"];
+  ADM_TYPE_ELECTIVE = 3  		  [(string_value) = "3"];
+  ADM_TYPE_NEWBORN = 4        [(string_value) = "4"];
+  ADM_TYPE_TRAUMA_CENTER = 5  [(string_value) = "5"];
+  ADM_TYPE_9 = 6 		          [(string_value) = "9"];
```

Comment:
This was a straight copy/pasta from the RDA source. Oddly enough it looks fine in my editor so it might be a tabs/spaces thing 
<img width="447" alt="image" src="https://github.com/CMSgov/beneficiary-fhir-data/assets/162160982/381e6578-21ef-474c-ba96-fa9a8616dfce">

Reply:
Apparently this is just an issue with the upstream proto files. Nothing to see here. Thanks!